target/linux/lantiq/files/arch/mips/boot/dts/lantiq/vr9_arcadyan_vrv9510kwac23.dts

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "vr9_arcadyan_vrv9510kwac23.dtsi"
```

Comment:
Do you plan to add more devices with the same HW?
If you do not plan to add more identical devices you can combine the dts and dtsi file into one.